Segments/Blackburrow.mapproj

```diff
@@ -269807,6 +269826,193 @@ return yima;
       <script name="OnIncomingPlayer" enabled="false">
         <block><![CDATA[]]></block>
         <block><![CDATA[
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+    </entity>
+    <entity name="BossNoteableLevel18InsaneWiz">
+      <script name="OnSpawn" enabled="true">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+var insaneWiz = new Humanoid()
+    {
+        Name = "Insane Froglok Wizard", Alignment = Alignment.Chaotic,
+        MaxHealth = 4500, Health = 4500,
+        BaseDodge = 40,
+        Body = 453,
+        CanSwim = true,
+    
+        Experience = 250000,
+        
+        BasePenetration = ShieldPenetration.Heavy,
+        HideDetection = 45,
+
+        Immunity  =  CreatureImmunity.Poison,
+        
+        
+        VisibilityDistance = 3,
+          
+    };
+    
+    insaneWiz.Stats[EntityStat.FireProtection].Base = 100;
+    insaneWiz.Stats[EntityStat.IceProtection].Base = 75;
+    insaneWiz.Stats[EntityStat.LightningResistance].Base = 75;
+    insaneWiz.AddImmunity(typeof(WhirlwindSpell));
+    insaneWiz.AddImmunity(typeof(PoisonCloudSpell));
+   
+    insaneWiz.AddStatus(new NightVisionStatus(insaneWiz));
+    insaneWiz.AddStatus(new BlindFearProtectionStatus(insaneWiz));
+    
+    insaneWiz.CombatantChangeInterval = TimeSpan.FromSeconds(3.0);
+    insaneWiz.CombatantSearchStrategy = (attacker) => new RandomSearchStrategy();
+    
+    insaneWiz.Equip(new DragonScaleArmor());
+    
+    insaneWiz.Attacks = new CreatureAttackCollection()
+    {
+        new CreatureBasicAttack(20, 30, 69, new AttackDazeComponent (10), new AttackProneComponent(5))
+    };
+    
+    insaneWiz.Spells = new CreatureSpellCollection(25, 75)
+    {
+       { new CreatureSpell<IceSpearSpell>(skillLevel: 12), 33, TimeSpan.FromSeconds(12.0) },
+       { new CreatureSpell<MagicMissileSpell>(skillLevel: 18), 33, TimeSpan.FromSeconds(6.0) },
+       { new CreatureSpell<CreateLavaSpell>(skillLevel: 9), 17, TimeSpan.FromSeconds(60.0)},
```

Comment:
As mentioned Lava to players might be a bit rough here. I see know it's 9, but pushing gear / etc if they die might be rough in recovery. I don't know if he will cast lava if the crit is on top of the player. It's pathed at least one hex to my knowledge. We had to put special logic for DB spells to work on same hex. Also due to the percentage (17) it is likely it won't be the first cast. 

Also other crits will be pushed / and it will bounce. I don't know what it does in water.